src/tracks/playabletrack/wavetrack/ui/PitchAndSpeedDialog.cpp

```diff
@@ -156,8 +156,8 @@ void PitchAndSpeedDialog::Destroy(AudacityProject& project)
 
 PitchAndSpeedDialog::PitchAndSpeedDialog(AudacityProject& project)
     : wxDialogWrapper(
-         nullptr, wxID_ANY, XO("Pitch and Speed"), wxDefaultPosition,
-         { 480, 250 }, wxDEFAULT_DIALOG_STYLE)
+         FindProjectFrame(&project), wxID_ANY, XO("Pitch and Speed"),
```

Comment:
I thought I did try that, but it led to some crash, can't remember the exact circumstances. Anyways, that seems to work now and I don't see by looking at the code why it shouldn't.